.clang-format

```diff
@@ -47,6 +47,7 @@ Cpp11BracedListStyle: true
 DerivePointerAlignment: false
 DisableFormat:   false
 ForEachMacros:   [ foreach, Q_FOREACH, BOOST_FOREACH ]
+IncludeBlocks: Regroup
```

Comment:
Did you intend to commit clang changes?

Reply:
Yes, this should reflect how the existing includes are handled a bit more closely.